src/Core/Components/Switch/FluentSwitch.razor.cs

```diff
@@ -0,0 +1,64 @@
+// ------------------------------------------------------------------------
+// MIT License - Copyright (c) Microsoft Corporation. All rights reserved.
+// ------------------------------------------------------------------------
+
+using System.Diagnostics.CodeAnalysis;
+using System.Reflection.Metadata;
+using Microsoft.AspNetCore.Components;
+using Microsoft.AspNetCore.Components.Web;
+
+namespace Microsoft.FluentUI.AspNetCore.Components;
+
+/// <summary>
+/// The FluentSwitch component is used to render a Switch
+/// </summary>
+public partial class FluentSwitch : FluentInputBase<bool>, IFluentComponentElementBase
```

Comment:
Why do you need to implement `IFluentComponentElementBase`. The `Element` parameter must be included only if needed (e.g. JS calls)